l3kernel/doc/l3prefixes.csv

```diff
@@ -163,6 +163,7 @@ pdfxform,l3pdfmanagement,The LaTeX Project,https://www.latex-project.org/latex3.
 peek,l3kernel,The LaTeX Project,https://www.latex-project.org/latex3.html,https://github.com/latex3/latex3.git,https://github.com/latex3/latex3/issues,2012-09-27,2012-09-27,
 percent,l3kernel,The LaTeX Project,https://www.latex-project.org/latex3.html,https://github.com/latex3/latex3.git,https://github.com/latex3/latex3/issues,2018-05-12,2018-05-12,
 pgf,pgf,The PGF/TikZ Team,https://pgf-tikz.github.io,https://github.com/pgf-tikz/pgf,https://github.com/pgf-tikz/pgf/issues,2020-07-03,2020-07-03,
+pgfmxfp,pgfmath-xfp,Jonathan P. Spratte,https://github.com/Skillmon/ltx_pgfmath-xfp,https://github.com/Skillmon/ltx_pgfmath-xfp,https://github.com/Skillmon/ltx_pgfmath-xfp/issues,2021-05-20,2021-05-19,
```

Comment:
Your `Last update` field is earlier than `First registered`. That seems wrong.

Reply:
I always assumed that `Last update` refers to the last update to the entry in `l3prefixes` and not the last update of the package.

@josephwright I'm guessing you created the file/column. Any insights regarding the intended meaning?

Reply:
That would be more reasonable (at least all my other entries don't reflect the latest package version).

Reply:
As we are not trying to keep track of package versions here, I think the  two columns have to be interpreted as: "when was this entry first registered" and "when was ithe entry last updated".

 the second is an (small) indicator of whether or not the data is likely to be current (in case there seem to be an issue, like not finding the listed home page, for example.